docs/nx-cloud/private/ami-setup.md

```diff
@@ -0,0 +1,143 @@
+   - [Or create a new one and download the keys locally](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html?icmpid=docs_ec2_console#having-ec2-create-your-key-pair)
+     - Then select your new SSH pair from the list
+5. Networking:
+   - Allow the instance to receive HTTP and HTTPS traffic
+   - Allow SSH from your current IP
+6. Leave the storage options as they are (external storage of 50GB)
+7. "Launch instance"
+8. Wait 10 minutes, then navigate to your instance's IP in the browser. You should see the NxCloud dashboard!
+
+### Your NxCloud URL
+
+1. At this point, you can consider your instance's IP address the URL of NxCloud.
+   - You can use NxCloud and all the below steps will work for you.
+2. You might want, however, to add a Load Balancer in front of the instance, with an explicit domain (e.g. https://my-nxcloud.my-org.com).
+   - This is strongly recommended because you will be able to upgrade/restart/re-configure your NxCloud EC2 instance while keeping the NxCloud URL static.
+   - Create an [application load balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/create-application-load-balancer.html)
+   - You will need to create a certificate for your domain to assign to the LB
+
+Once you have your NxCloud URL proceed to the below steps!
+
+### Configuring your NxCloud instance
+
+1. Create a new `myconfiguration.yaml` file with the below contents
+
+```yaml
+# This is all you need to get the baseline of your nx-cloud instance configured!
+
+# Set the external URL your instance is running on. This is the URL from the previous step
+nxCloudAppURL: 'https://nx-cloud.on.my-domain.ca'
+
+secret:
+  # set your initial admin password for logging into the app
+  adminPassword: 'correcthorsebatterystaple'
+```
+
+2. Apply the configuration:
+
+```bash
+scp -r ./myconfiguration.yaml nx-cloud@<your-instance-ip>:~/config/user/update.yaml
+```
+
+That's it! After a few minutes, you should be able to log-in using your password!
+
+### Connecting to your NxCloud instance
+
+In your Nx workspace, you can enable NxCloud by running:
+
+```bash
+NX_CLOUD_API="https://nx-cloud.on.my-domain.ca" npx nx connect
+```
+
+### Advanced configuration and auth
+
+You can optionally enable authentication using your preferred SSO provider:
+
+- GitHub
+- Bitbucket
+- GitLab
+- SAML (Okta, Azure AD etc.)
+
+```yaml
+# This is all you need to get the baseline of your nx-cloud instance configured!
+
+# only use this if you'd like to use any of the newer NxCloud version from here: https://nx.dev/nx-cloud/reference/release-notes#docker-containers
+# global.imageTag: ''
+
+# Set the external URL your instance is running on
+nxCloudAppURL: 'https://nx-cloud.on.my-domain.ca'
+
+# Uncomment (along with github secrets below) to enable working with GitHub pull requests or github auth
+#github:
+#  auth:
+#    enabled: false
+
+#gitlab:
+#  apiUrl: '' # this is only needed if you have a self-hosted instance
```

Comment:
might be just _me_ but I thought this was for is you're doing self hosting nx cloud, not self hosted gitlab.
```suggestion
#  apiUrl: '' # this is only needed if you have a self-hosted gitlab instance
```

Reply:
yes, you're right! I'll fix the wording!